package/etc/conf.d/conflib/syslog/app-syslog-liveaction_livenx.conf

```diff
@@ -0,0 +1,18 @@
+block parser app-syslog-liveaction_livenx() {
+    channel {
+        rewrite {
+            r_set_splunk_dest_default(
+                index("netops")
+                sourcetype('liveaction:livenx')
+                vendor("liveaction")
+                product("livenx")
+            )
+        };
+    };
+};
+application app-syslog-liveaction_livenx[sc4s-syslog] {
+    filter {
+        message('%LIVEACTION'  type(string)  flags(prefix));
```

Comment:
program('<string to use>' type(string) flags(prefix))

Reply:
Message is not considering `%LIVEACTION` as program and hence I have parsed it as message.